client/src/components/browseRecipes/Category.svelte

```diff
@@ -0,0 +1,94 @@
+<script>
+  import { createEventDispatcher, onMount } from 'svelte';
+  import { categoriesApi } from '@/api';
+  const dispatch = createEventDispatcher();
+
+  function handleChange({ target }) {
+    dispatch('categoryToggle', {
```

Comment:
can you rename this dispatch event since it is not a toggle but a change or update. I would name it just `update`, category part is not needed since we are in the context of categories